packages/react/src/components/Input/index.stories.js

```diff
@@ -51,6 +54,7 @@ stories.add('Playground', () => (
     invalid={boolean('Is invalid?', false)}
     valid={boolean('Is valid?', false)}
     disabled={boolean('Is disabled?', false)}
+    inputIcon={select('Input icon', ICONS, ICONS.None)}
```

Comment:
I'd go with `icon` 
```suggestion
    icon={select('Icon', ICONS, ICONS.None)}
```
Because `<Input inputIcon="x" />` feels a bit repetitive
Where `<Input icon="x" feels more natural IMO

Reply:
Dang that was my first choice!!! :-D
Changed!